.semgrep/adapter/builder-struct-name.yml

```diff
@@ -3,9 +3,22 @@ rules:
     languages:
       - go
     message: >
-      Scope of `$BUILDER` is limited to this adapter package. Therefore `$BUILDER` can be renamed to `adapter`. Refer to the following example.
+      Using the adapter name for the struct makes it redundant with the package name. 
```

Comment:
I would recommend making this comment a little simpler such as:
```
Using $BUILDER for this struct name makes it redundant with the package name. Consider renaming this struct to "adapter". See example below:
```

Reply:
https://github.com/prebid/prebid-server/pull/2833/commits/bd7aaffb9104aeda884a420f785bbdcef74786c3

thanks for the updated message